### PR TITLE
Update Latin translation

### PR DESCRIPTION
**Pull request by sls1005**
I changed the word for "movement" from *motus* to *momentum* because *motus* is a perfect participle, similiar to "moved" in English. It can mean "motion" in Latin but seems to be far from the idea of "the thing by which it moves," "the ability or potential to move" or "that which is taken (consumed) for it to move." So I picked the word "momentum," which has the suffix "-mentum," which can mean "instrument" or "medium" and finally becomes "-ment" in English.

TLDR:
Movement = Motus (when referrimg to motion or the action of moving) / Momentum (when refering to the ability or potential to move)

Thus, "movement cost" should be something like *momentum sumptum* ("move-ment (tool or something) that is taken (consumed)").

### CODE REVIEW

File: android/assets/jsons/translations/Latin.properties [hunk 1]
```diff
@@ -11310,19 +11318,19 @@ Machine Gun = Polybolum
 
 
  # Requires translation!
-Landship = 
+Landship = Navis Terræ
```
Comment (yairm210) [inline android/assets/jsons/translations/Latin.properties:11321]: Nice ;)
